gdal/frmts/netcdf/netcdfdataset.cpp

```diff
@@ -3160,6 +3160,17 @@ void netCDFDataset::SetProjectionFromVar( int nVarId, bool bReadSRSOnly )
             }
 
 
+            // If the data is bottom up then flip the y - coords
+            // this is significantly faster than reading row by row
+            if (poDS->bBottomUp)
```

Comment:
Hmm, so I don't care so much about this as a "feature" but more as an optimization.

So, do you have any thoughts on how to remove code like https://github.com/GlobalPixels/gdal/blob/d4a21b501bdda66edf2d5eb239a21c23167a32ad/gdal/frmts/netcdf/netcdfdataset.cpp#L510 and https://github.com/GlobalPixels/gdal/blob/d4a21b501bdda66edf2d5eb239a21c23167a32ad/gdal/frmts/netcdf/netcdfdataset.cpp#L1440

Reply:
I haven't really looked at that myself, so I don't pretend to have an informed opinion. But I'm not sure why you couldn't use a larger block height than 1. That would require a bit of extra work because you'd need to flip lines once you've read them with nc_get_vara_XXXXX().